docker-compose.yaml

```diff
@@ -20,6 +20,7 @@ services:
       - ${PWD}:/code/
     ports:
       - 127.0.0.1:8081:8081
+      - 127.0.0.1:8080:8080
```

Comment:
why this?

Reply:
Oopsies, was easier to test with it :slightly_smiling_face: